en/core-api/e-wallet.md

```diff
@@ -302,49 +351,73 @@ You will get the **API response** like the following.
   ]
 }
 ```
-You will get the `actions` attribute which can be performed this transaction.
+You will get the `actions` attribute to complete the transaction.
 
-## 2. Alter payment flow depending on the device used
+<details>
+<summary><b>Response Body</b></summary>
+<article>
+
+| Element            | Description                                                  | Type   | Notes                                                        |
+| ------------------ | ------------------------------------------------------------ | ------ | ------------------------------------------------------------ |
+| status_code        | The status of the API call.                                  | String | For more details, refer to [Status Codes and Error](/en/technical-reference/error-response-code.md#status-codes-and-errors). |
+| status_message     | A message describing the status of the transaction.          | String | --                                                           |
+| transaction_id     | The *Transaction ID* of the specific transaction.            | String | --                                                           |
+| order_id           | The specific *Order ID*.                                     | String | --                                                           |
+| gross_amount       | The total amount of transaction for the specific order.      | String | --                                                           |
+| currency           | The unit of currency used for the transaction.               | String | --                                                           |
+| payment_type       | The type of payment method used by the customer for the transaction. | String | --                                                           |
+| transaction_time   | The date and time at which the transaction occurred.         | String | It is in the format, *YYYY-MM-DD* *HH:MM:SS.*<br>Time zone: Western Indonesian Time (GMT+7) |
+| transaction_status | The status of the transaction.                               | String | For more details, refer to [Transaction Status](/en/after-payment/get-status.md#transaction-status). |
+| fraud_status       | The fraud_status of the transaction is displayed.            | String | --                                                           |
+| actions            | The stage of the transaction                                 | Array  | There are four stages of the transaction that can be retrieved through this attribute: QR code, Deeplink, Status of transaction, Canceling the transaction |
```

Comment:
> The stage of the transaction

>There are four stages of the transaction that can be retrieved through this attribute: QR code, Deeplink, Status of transaction, Canceling the transaction

It is not stage, it is a set of actions that can be performed by merchant, they choose which action to perform according to their needs.

So it is not sequential/stages.